analysis/tensorflow/fast_em.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Wrapper to run fast_em.py using TensorFlow configured for a GPU.  CUDA
+# environment variables must be set.
+#
+# Usage:
+#   ./fast_em.sh <args>
+
+set -o nounset
+set -o pipefail
+set -o errexit
+
+readonly THIS_DIR=$(dirname $0)
+
+fast-em() {
+  export LD_LIBRARY_PATH=/usr/local/cuda/lib64
+  export CUDA_HOME=/usr/local/cuda-7.0
+
+  # Never returns
+  exec $THIS_DIR/fast_em.py "$@"
```

Comment:
You shouldn't clobber the existing variable for LD_LIBRARY_PATH rather you should append that to it. 
Also you don't need to export, you can just assign the variables on the same line as the exec and the child will inherent them without messing with them in the user's shell afterwards.